tests/scripts/thread-cert/border_router/test_advertising_proxy.py

```diff
@@ -174,7 +174,7 @@ def test(self):
         self.assertIsNone(host.discover_mdns_service('my-service-1', '_ipps._tcp', 'my-host'))
 
         server.srp_server_set_enabled(True)
-        self.simulator.go(LEASE)
+        self.simulator.go(LEASE + 5)
```

Comment:
Thanks @librasungirl. 
- I submitted this very similar PR https://github.com/openthread/openthread/pull/10474 earlier  :)

Reply:
PR #10474 also addresses the following:

> - Due to the use of short lease time (10 seconds) in this test, the client will refresh the registered services quickly. Therefore, in `check_host_and_service()`, any of `Registered`, `ToRefresh`, or `Refreshing` states are accepted as indicating successful registration.

I don't think the use of `go(LEASE)` here is good, since the wait time here isn't related to the `LEASE` interval. The delay here is after the server is re-enabled and we're waiting for the client to re-register its services.

Reply:
PR https://github.com/openthread/openthread/pull/10474 is a more considerate solution. will close this one.